dev/bender/plugins/ckeditor5/static/extensions.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals bender, CKEDITOR */
+
+'use strict';
+
+( function() {
+	// Override bender.start to wait until the "ckeditor" module is ready.
+	var originalStart = bender.start;
+	bender.start = function() {
```

Comment:
I'm not sure if we discussed this already, but I think we should consider adding some space between the variable definition and the rest of the code. It would improve the readability of the code because right now it looks a bit crammed.